angular-dfp.js

```diff
@@ -439,3 +439,4 @@ angular.module('ngDfp', [])
     };
   }]);
  
+module.exports = 'ngDfp';
```

Comment:
Sorry Ian, your are correct. I have now removed this change and implemented a better way of doing it, utilising the npm way of handling modules.

I have added an index.js file that requires the angular-dfp.js script and exports the module name. The index file is referenced in the package.json file as the main file. This allows any npm/common.js users to implement the angular-dfp module as an npm/commonjs module but it also allows non npm/commonjs users to use the file in the standard way by referencing the angular-dfp scipt in html.